gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -7630,7 +7628,6 @@ static PyObject* GemRB_ChangeStoreItem(PyObject * /*self*/, PyObject* args)
 	}
 	default: break;
 	}
-#pragma GCC diagnostic pop
```

Comment:
Apparently not in gcc7. Official docs mention default should circumvent it, but it was probably added later. You can just add `-Wno-switch-enum` to CXXFLAGS.

Perhaps -Wall includes both, but then I'd expect the error to mention it:
https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wswitch

Reply:
we could, but in general I feel like thats something we want.

Reply:
Why? If there's no default case, it will still warn (plus Sonar already complains).

Reply:
I was thinking extraneous cases, but yeah, sonar is probably sufficient.

Reply:
@lynxlynxlynx did I not do that right? linux build is failing now.

Reply:
meh, I just refactored to split the two things